### DISCUSSION TITLE
I was very slow to start and discovered this issue

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12547] [Q&A]
https://github.com/AvaloniaUI/Avalonia/assets/53590491/b9306b0b-a13e-4371-87ef-71261694131f

**Comment by wosiyourfather** [comment]
![0](https://github.com/AvaloniaUI/Avalonia/assets/53590491/cb3c318b-ac50-442b-9408-de5ba015d9b3)

**Comment by robloo** [comment]
It's not great to just attach videos and images to a discussion/issue like this. We are unable to translate without a lot of work. I recommend providing more information in text. If you can in both your native language and English side-by-side.

Otherwise, you are going to have to wait until another Avalonia user that can read your image helps.

**Reply by kyuranger** [reply]
He installed a non-English version Visual Studio and the default language of his Windows is non-English also. He has to reinstall an English Visual Studio language package and set it as default language. Moreover, he needs to set the default culture in the project to English.

**Comment by kyuranger** [comment]
Does the same happen when you create a brand-new Avalonia project? Which version of the Avalonia framework are you using?
